### ISSUE TITLE
Pioneer Community Roadmap

### ISSUE THREAD

**Issue by traumschule** [Joystream/pioneer#4435]
[JSG Priorities](https://github.com/Joystream/pioneer/issues/4106)
[Design](https://github.com/Joystream/pioneer/issues/2178)

### Q2
- [Election issues](https://github.com/Joystream/pioneer/issues/3737)
- [Block Explorer](https://github.com/Joystream/pioneer/issues/4367)
- [Validators](https://github.com/Joystream/pioneer/issues/4321) - [Val Email](https://github.com/Joystream/pioneer/issues/4024)
- [Notifications](https://github.com/Joystream/pioneer/issues/1925) -> [Pioneer Notifications](https://github.com/Joystream/pioneer/issues/4196) -> [Email Notifications](https://github.com/Joystream/pioneer/issues/4230)
- [Proposal Module](https://github.com/Joystream/pioneer/issues/3783)

### Q3
- [Bounties v1 bugs](https://github.com/Joystream/pioneer/issues/3519) [Bounties v2](https://github.com/Joystream/pioneer/issues/2883)
- [Council Enhancements](https://github.com/Joystream/pioneer/issues/1924) [Calendar](https://github.com/Joystream/pioneer/issues/51) [Financials](https://github.com/Joystream/pioneer/issues/46)
- [WG](https://github.com/Joystream/pioneer/issues/1921) [WG Roles](https://github.com/Joystream/pioneer/issues/3772) [Content Payout](https://github.com/Joystream/pioneer/issues/2018) [Forum](https://github.com/Joystream/pioneer/issues/1919)

### Q4
- [Responsive design](https://github.com/Joystream/pioneer/issues/1794)
- [Tech Debt: TODO](https://github.com/Joystream/pioneer/issues/3985) [QN Subsquid Migration](https://github.com/Joystream/pioneer/issues/2036)
- UX: [loading states](https://github.com/Joystream/pioneer/issues/1214) [UI Tests](https://github.com/Joystream/pioneer/issues/3521) [UX Upd](https://github.com/Joystream/pioneer/issues/3030)
- [Support & Enhancements](https://github.com/Joystream/pioneer/issues/3031)

### 2024
- [Gateway UI](https://github.com/Joystream/pioneer/issues/3512)
- devops: [1-click deployment](https://github.com/Joystream/pioneer/issues/1783)
- [avatar storage](https://github.com/Joystream/pioneer/issues/54)
- [l18n](https://github.com/Joystream/pioneer/issues/1786)

**Comment by dmtrjsg** [comment]
Thank you @traumschule for putting this together, looking great! Just a few comments from my side to the major items (leaving support issues aside as these will be going in the background subject to capacity of WG and prioritisation from the lead):

Q2
⚠️ This overall block is now looking more likely to take a good part of Q3 for implementation.

**Block explorer** - I know that Chaos is looking into getting the JOY tokens in circulation values, and perhaps account balances as well, but not sure. We've already added link to it from Pioneer app, which is a good first step :)
**Validators** - Designs are in the final stage but likely to take a few more weeks based on the recent pace. Implementation would probably take another 4 weeks or so.
**Notifications**- FE part has not started yet, so likely to take good part of Q3 as well.

Q4
**Responsive design** - more likely candidate than others for Q4 since design is in progress now and nearing completion over next few weeks
**Financial module* - it is pretty big, so this and responsive may as well occupy the majority of Q4.
**Bounties** - there's a very substantial part of QN required and runtime upgrade required to enable this. Unlikely to fit in this year unless prioritised above the other two.

**Comment by dmtrjsg** [comment]
@traumschule can this issue be closed now?

**Comment by traumschule** [comment]
I don't think it should be closed, it's a meta issue and can be updated over time. Feel free!
